Bottom: 2a5df2da05eb7a45c693556cc7360209b44768e2
Top:    2568c9d198586363b577efd5fc8dad910be2a2f7
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-17 08:54:51 -0500

Create certificate directories during image build



---
diff --git a/docker/docker-mantisbt/configure.sh b/docker/docker-mantisbt/configure.sh
index 711b554f..a8e08843 100755
--- a/docker/docker-mantisbt/configure.sh
+++ b/docker/docker-mantisbt/configure.sh
@@ -1,6 +1,11 @@
 #!/bin/bash
 set -e
 
+# Create certificate directories
+mkdir -p /etc/grid-security/certificates
+mkdir -p /etc/ssl/certs/
+mkdir -p /etc/ssl/private
+
 # comment out apache2 config file lines that reference the environment variables
 sed -i 's|^Mutex file|#Mutex file|' /etc/apache2/apache2.conf
 sed -i 's|^PidFile |#PidFile |' /etc/apache2/apache2.conf
